### ISSUE TITLE
How  & where the results get saved ?

### ISSUE THREAD

**Issue by vikramvi** [Verkurkie/timings-docker#6]
Please clarify where does these results get saved and how do they get saved.

I was running containers on local ubuntu laptop, locked and after logging back found that Kibana not showing any graphs ( which were present earlier )

I checked the container status and found all are up & running.

**Comment by Verkurkie** [comment]
The timings API saves all the results in elasticsearch - if you had charts prior but don't see data anymore, try expanding the time-range in Kibana (in the upper-right corner). If you don't run/submit any tests, the results may disappear from the radar if the time-range is too small ...

**Comment by vikramvi** [comment]
Sorry let me rephrase my queries as below

- I understand that results are saved in elasticsearch but once that container goes down, will the results get lost ? or saved in local db ?
- How do you maintain this whole setup ?
- Container are not guaranteed to up always and may go down for some reason. Lets say elasticsearch container goes down, what will happen to data ?

**Comment by vikramvi** [comment]
ping @Verkurkie  

can you please review and clarify

**Comment by Verkurkie** [comment]
Vikram - to answer your questions:
* the API currently has no way of saving results when elasticsearch is down. it will still do the performance comparison but since it can't get a baseline, it will compare against the "flat SLA" that you provide in the POST payload (the `sla` key)
* If you're referring to elasticsearch, I am using an enterprise cluster that is maintained by a dedicated team. The data is also forked to Hadoop so i can easily rebuild the elasticsearch indices in case of an elasticsearch failure.
* See ^^^

Does that help?

**Comment by vikramvi** [comment]
Thanks for clarification, closing the issue

**Comment by vikramvi** [comment]
updated in https://github.com/Verkurkie/timings-docker/wiki/FAQ-page
